src/site/_includes/show.njk

```diff
@@ -1,18 +1,24 @@
 ---
-layout: default
+layout: default-next
 ---
 {% set data = paged.data %}
-<div class="w-layout-container--narrow w-post-content">
-  <header class="w-article-header">
-    <h1 class="w-article-header__headline">{{ data.title }}</h1>
-    <div class="w-author__published">
-      <time>{{ data.date | prettyDate }}</time>
-    </div>
-    <div class="w-layout-container--narrow w-post-signpost">
-      <span class="w-post-signpost__title">{{ 'i18n.post.appears_in' | i18n(locale) }}:</span>
-        <a class="w-post-signpost__link" href="{{ data.parent.url }}">{{ data.parent.title | i18n(locale) }}</a >
-    </div>
-  </header>
-  {% YouTube data.videoId %}
-  <p>{{ data.description | md({linkify: true, breaks: true}) | safe }}</p>
+<div class="post wrapper region" data-flush>
+  <article class="prose legacy-rollout">
+    <header class="flow gap-bottom-size-2">
+      <h1>{{ data.title }}</h1>
+      <div class="flow-space-base color-mid-text text-size-0">
+        <time>{{ data.date | prettyDate }}</time>
+      </div>
+    </header>
+
+    {% YouTube data.videoId %}
+    <p>{{ data.description | md({linkify: true, breaks: true}) | safe }}</p>
+    <p>
+      <a href="{{ data.parent.url }}" class="button" data-type="secondary">
+        {% include 'icons/arrow-back.svg' %}
+        <span>Back to all episodes</span>
```

Comment:
Should this be an i18n string? Not a deal breaker cause we dont generate shows in other languages, but just a though,

Reply:
I guess it wouldn't hurt, so I might as well do it :)